packages/wonder-blocks-tooltip/components/tooltip.js

```diff
@@ -1,22 +1,27 @@
 // @flow
 import * as React from "react";
 
-import {Text as WbText} from "@khanacademy/wonder-blocks-core";
+import {Text} from "@khanacademy/wonder-blocks-core";
```

Comment:
Use something from `wonder-blocks-typography` instead.

Reply:
See my comment in the other PR. I don't want layout nor styling to be changed by this, hence the use of `Text`.

Reply:
Sounds like the use of `Text` will eventually be replaced by something else in a future diff.